src/app/overview/home/home.component.ts

```diff
@@ -108,17 +110,20 @@ export class HomeComponent implements OnDestroy, OnInit {
         this.loadingTodoItems = false;
       })
     );
-    // only get the number of chats if user is in team
-    if (this.storage.getUser().teamId) {
-      this.subscriptions.push(
-        this.homeService.getChatMessage().subscribe(chatMessage => {
-          if (!this.utils.isEmpty(chatMessage)) {
-            this._addChatTodoItem(chatMessage);
-          }
-          this.loadingTodoItems = false;
-        })
-      );
-    }
+    // only get the number of chats if user has chatroom
+    this.chatService.getChatList().subscribe(chats => {
```

Comment:
we don't need to check this here. Previously we check if user is in team because if user is not in team, we can save the time to send an API request, since "check if user is in team" doesn't need an API request.
Now it's unnecessary to make an additional API request to check if user has channels. We can just send the request to get chat messages